examples/worlds/mimic_fast_slow_pendulums_world.sdf

```diff
@@ -0,0 +1,655 @@
+<?xml version="1.0" ?>
+<!--
+  Gazebo Mimic constraint demo
```

Comment:
Since this currently only works with bullet-featherstone, can you provide the command to run here?

Reply:
I added the example command in e9b22fcc76a510fd0b31395c81ddbc2a16ad69e2

I noticed that lots of our other example worlds are using command-line strings with `--` inside XML comment blocks, which is invalid XML, so I put the example command inside a CDATA block. Now `xmllint` is happy with the file, and I believe it still works.